src/site/content/en/blog/optimize-ttfb/index.md

```diff
@@ -161,18 +180,34 @@ CDN providers may also offer benefits beyond edge servers:
 
 While adopting a CDN involves a varying amount of effort from trivial to significant, it should be a high priority to pursue in optimizing your TTFB if your website is not already using one.
 
+### Used cached content where possible
+
+CDNs allow content to be cached at edge nodes, closer to visitors, if the content is configured with the appropriate [`Cache-Control` HTTP headers](https://developer.mozilla.org/docs/Web/HTTP/Headers/Cache-Control). While this is not appropriate for personalised content, requiring a trip all the way back to the origin can negate much of the value of a CDN.
+
+For sites that frequently update their content, even a short caching time can result in noticable performance gains for busy sites as only the first visitor during that time pays the full price, while all other visitors can use the cached resource. Some CDNs allow cache invalidation on site releases allowing the best of both worlds—long cache times, but insteant updates when needed.
+
+Even where caching is correctly configured, this can be ignored through the use of unique query params for analytics measurement. These may look like different content to the CDN despite being the same, and so the cached version will not be used.
+
+Older, or less visited content, may also not be cached which can result in higher TTFBs on some pages than others. Increasing caching times can reduce the impact of this.
+
+The impact of cached content does not just affect those using CDNs. Server infrastructure may need to generate content from costly database lookups when cached content cannot be reused. More frequently accessed data, or pre-cached pages can often perform better.
+
 ### Avoid multiple page redirects
 
-One potential contributor to a high TTFB is [redirects](https://developer.mozilla.org/docs/Web/HTTP/Redirections). Redirects occur when a navigation request for a document receives a response that informs the browser that the resource exists at another location. One redirect can certainly add unwanted latency to a navigation request, but it can certainly get worse if that redirect points to another resource that results in _another_ redirect—and so on.
+One common contributor to a high TTFB is [redirects](https://developer.mozilla.org/docs/Web/HTTP/Redirections). Redirects occur when a navigation request for a document receives a response that informs the browser that the resource exists at another location. One redirect can certainly add unwanted latency to a navigation request, but it can certainly get worse if that redirect points to another resource that results in _another_ redirect—and so on.
+
+This can particularly impact sites that receive high volumes of visitors from advertisements or newsletters, since they often redirect via analytics services for measurement purposes.
+
+Redirect time, is a direct reduction in remaining time to achieve a good LCP.
```

Comment:
```suggestion
One common contributor to a high TTFB is [redirects](https://developer.mozilla.org/docs/Web/HTTP/Redirections). Redirects occur when a navigation request for a document receives a response that informs the browser that the resource exists at another location. One redirect can certainly add unwanted latency to a navigation request, but it can certainly get worse if that redirect points to another resource that results in _another_ redirect—and so on. This can particularly impact sites that receive high volumes of visitors from advertisements or newsletters, since they often redirect via analytics services for measurement purposes. Eliminating redirects under your direct control can help to achieve a good LCP.
```